### PR TITLE
chore: lint the incentives module

### PR DESCRIPTION
**Pull request by faddat**
## What is the purpose of the change

This PR lints the incentives module in osmosis. 

## Testing and Verifying

This change is a trivial rework / code cleanup without any test coverage.

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes? no
  - [ ] Changelog entry added to `Unreleased` section of `CHANGELOG.md`? no

Where is the change documented? 
  - [ ] Specification (`x/{module}/README.md`)
  - [ ] Osmosis documentation site
  - [ ] Code comments?
  - [x] N/A

### CODE REVIEW

File: x/incentives/keeper/distribute_test.go [hunk 1]
```diff
@@ -369,19 +369,18 @@ func (s *KeeperTestSuite) TestDistribute_ExternalIncentives_NoLock() {
...
+		distrTo           lockuptypes.QueryCondition
+		startTime         time.Time
+		numEpochsPaidOver uint64
+		poolId            uint64
 
 		// expected
 		expectErr                              bool
 		expectedDistributions                  sdk.Coins
-		expectedRemainingAmountIncentiveRecord []sdk.Dec
+		expectedRemainingAmountIncentiveRecord []sdkmath.LegacyDec
```
Comment (pysel) [inline x/incentives/keeper/distribute_test.go:383]: nit: shall we use `osmomath.Dec` here?

Comment (pysel) [reply x/incentives/keeper/distribute_test.go:383]: https://github.com/osmosis-labs/osmosis/pull/6288/files#diff-d5967ddef40ae47ef4140b9c47026cac85b85d9d04173b271c6c23dd182abdbcR408

https://github.com/osmosis-labs/osmosis/pull/6288/files#diff-d5967ddef40ae47ef4140b9c47026cac85b85d9d04173b271c6c23dd182abdbcR425

and etc

**Review by pysel — COMMENTED** [review]
nice! just one nit

File: x/incentives/keeper/distribute_test.go (hunk 1 above)
Comment (faddat) [reply x/incentives/keeper/distribute_test.go:383]: It is a very good nit, and I wasn't sure. 

@p0mvn has been moving things over to use cosmossdk.io/math but I think this is a great question to bring up.  Thanks!

Comment (faddat) [reply x/incentives/keeper/distribute_test.go:383]: (let's not merge this till there's an answer to this question)

**Comment by faddat** [comment]
closing

**Pull request closed without merging**